roles/openshift_node_dnsmasq/tasks/main.yml

```diff
@@ -0,0 +1,33 @@
+---
+- name: Set requisite facts
+  openshift_facts:
+    role: common
+    local_facts:
+      portal_net: "{{ openshift_portal_net | default(openshift_master_portal_net) | default(None) }}"
```

Comment:
We should be relying on this being set from openshift_facts (or openshift_common) rather than setting it here.